### DISCUSSION TITLE
Trouble running Hello world

### DISCUSSION THREAD

**Discussion by Srinivas-Ganti** [NanoComp/meep#2128] [Q&A] [answered]
Hi all,

I am trying to follow along with the documentation and tried to run some of the example code to see what output I get on my machine. I tried to run the 'Hello world' example given [here](https://meep.readthedocs.io/en/latest/The_Run_Function_Is_Not_A_Loop/):

I get this error:

TypeError: __init__() missing 1 required positional argument: 'resolution'

Any idea what might be going on? I have Meep installed, I can import the module, material parameters like SiO2 example given a few pages earlier in the documentation.

**Comment by Srinivas-Ganti** [comment]
Update: I am able to run the next example given [here](https://github.com/NanoComp/meep/blob/master/python/examples/refl-angular-kz2d.py) so it seems Meep is installed correctly and working. Just unsure what output was expected in the simple Hello world case and why it didn't work for me..

**Comment by stevengj** [comment, marked as answer]
I'm not sure exactly what you are running, but that page is not really intended as your first tutorial, it's about how to pass "step functions" to the `mp.run` function.   In particular, the first example `sim = meep.Simulation(...)` is not literally runnable — you are supposed to fill in the `...` with whatever arguments you want of the simulation.

Instead, I would start with the [Basic Tutorial](https://meep.readthedocs.io/en/latest/Python_Tutorials/Basics/).

**Reply by Srinivas-Ganti** [reply]
Thank you for clarifying, I am following along with the examples :)
